libs/SalesforceAnalytics/SalesforceAnalyticsTests/EventStoreManagerTests.m

```diff
@@ -105,23 +105,6 @@ - (void) testFetchOneEvent {
 }
 
 /**
- * Test for fetching all stored events.
- */
-- (void) testFetchAllEvents {
```

Comment:
This test is same as testStoreMultipleEvents, so I removed this one, and rename testStoreMultipleEvents to testStoreAndFetchMultipleEvents